android/gradle.properties

```diff
@@ -23,8 +23,7 @@ kotlin.code.style=official
 android.nonTransitiveRClass=true
 
 GROUP=com.revenuecat.purchases
-VERSION_NAME=8.1.2
-POM_DESCRIPTION=Mobile subscriptions in hours, not months.
+VERSION_NAME=8.1.3-beta.5POM_DESCRIPTION=Mobile subscriptions in hours, not months.
```

Comment:
Hmm this is weird... I just executed `bundle exec fastlane bump`... Will take a look at this.

Reply:
I think it's because the .version lost EOFL line in the file? Take a look at it in the diff